server/service/src/sync/translations/name.rs

```diff
@@ -41,17 +38,17 @@ impl LegacyNameType {
         }
     }
 
-    pub fn from_name_type(name_type: &NameType) -> Self {
-        match name_type {
-            NameType::Facility => LegacyNameType::Facility,
-            NameType::Patient => LegacyNameType::Patient,
-            NameType::Build => LegacyNameType::Build,
-            NameType::Invad => LegacyNameType::Invad,
-            NameType::Repack => LegacyNameType::Repack,
-            NameType::Store => LegacyNameType::Store,
-            NameType::Others => LegacyNameType::Others,
-        }
-    }
+    // pub fn from_name_type(name_type: &NameType) -> Self {
```

Comment:
We are not yet planning to use this, didn't want to add #[allow attribute, since code could be `dead` for ever

Reply:
Nope, didn't we agree that documents are pushed, and other sites deduce the state of name based on latest document ? I could be missing something